toolkit/cli/smart-contracts-commands/src/governance.rs

```diff
@@ -58,9 +62,15 @@ impl InitGovernanceCmd {
 pub struct UpdateGovernanceCmd {
 	#[clap(flatten)]
 	common_arguments: crate::CommonArguments,
+	/// Old governance authority hash to be set.
+	#[arg(short = 'o', long, num_args = 1.., value_delimiter = ' ')]
+	old_governance_authority: Vec<MainchainKeyHash>,
```

Comment:
Does order matter from user point of view?
If yes, we should sort both old and new authorities before passing them to the offchain code (or sort them there)